.gitignore

```diff
@@ -7,3 +7,6 @@ dist
 /*.prql
 */*.prql
 target
+node_modules/
+package.json
+package-lock.json
```

Comment:
Surely it's quite important these two `.json` files aren't ignored? They're key files in npm projects...

Reply:
I believe these are artifacts of installing Prettier.  At least the `package.json` contains the following. I'm not sure why they showed up so recently (as I was re-running all the installation tests last night prior to committing the most recent files.)

```
{
  "dependencies": {
    "prettier": "^2.7.1",
    "prettier-plugin-go-template": "^0.0.13"
  }
}
```

Reply:
If we can repro that, we should resolve it. But those are important files elsewhere and shouldn't be in the `.gitignore` for sure

Reply:
Those files have never been present in the top-level prql/prql main branch. I don't know where they came from, but I'll delete them from the PR, as well as removing them from `.gitignore`. Then if they re-appear, we can be attentive...

Reply:
Great. (FYI having them in the gitignore without a `/` prefix means they're ignored everywhere, not just the top level)